src/index.ts

```diff
@@ -20,6 +20,8 @@ import {
   type ThemeInput,
   type NodeData,
   type JsonData,
+  type TypeFilterFunction
```

Comment:
```suggestion
  type TypeFilterFunction,
```

Looks like a missing comma there ;)